XenAdmin/Controls/MultipleDvdIsoList.cs

```diff
@@ -51,6 +51,8 @@ public partial class MultipleDvdIsoList : UserControl
         public MultipleDvdIsoList()
         {
             InitializeComponent();
+            labelSingleDvd.ForeColor = Program.HeaderGradientForeColor;

```

Comment:
By doing this, the ForeColor for these labels will be set to HeaderGradientForeColor (white) for all usages of the MultipleDvdIsoList control, including one on VM Storage page, where the background is also white.